### ISSUE TITLE
HTML rendering of ScoreMatrix doesn't work if models or tests are duplicated

### ISSUE THREAD

**Issue by rgerkin** [scidash/sciunit#82]
If the ScoreMatrix has duplicate models or tests, e.g.:  
```
suite = sciunit.TestSuite([test,test])
sm = suite.judge([model,model,model])
```
Then the score matrix will not render correctly in Jupyter because when it tries to extract one column or row it ends up extracting a DataFrame instead because there are multiple matching values (and hence no reduction of dimensionality).

**Comment by ChihweiLHBird** [comment]
In `dev` branch, this can be solved by adding 
```
if isinstance(score, pd.Series) and score.ndim == 1:
    score = score[0]
elif isinstance(score, pd.DataFrame) and score.ndim == 2:
    score = next(score.items(), None)[1][0]
```
after line #425, outside of the else statement.
Another possible solution is raise an exception and prohibit creation of a ScoreMatrix object that contains duplicated models or tests.

**Comment by ChihweiLHBird** [comment]
Fixed in 8da1cbb7ace0d3088e6770215a7e057dd4c1d7cc
